### DISCUSSION TITLE
how do I use this CLΩ and CL5?

### DISCUSSION THREAD

**Discussion by Colin1248** [cloudlink-omega/.github#2] [Q&A]
Hello I cannot figure out how to use these In a game I am making and the wiki does not help(well it does but not enough)
help please can someone make a sample project or something I cant figure it out AT ALL.

**Comment by nmsderp** [comment]
> Hello I cannot figure out how to use these In a game I am making and the wiki does not help(well it does but not enough) help please can someone make a sample project or something I cant figure it out AT ALL.

its not finished so it won't work well

**Reply by Colin1248** [reply]
oh ok

**Reply by glkdrlgkrlzflnjkgh** [reply]
hi, how do I even use this version anyway, how do I installit to turbowarp?

**Reply by Drago-Cuven** [reply]
> hi, how do I even use this version anyway, how do I installit to turbowarp?

dude. -_- the fact that you don't know this and yet you're trying to use "this" extension is wild. i suggest getting involved with the [actual turbowarp server](https://discord.gg/jVBpHyht). in the extensions section, there is literally a BIG OL BUTTON for putting in custom extensions.
![image](https://github.com/user-attachments/assets/8a1ca556-7035-4db1-84c3-9e14733d0bed)

**Reply by Thebloxers998** [reply]
> > hi, how do I even use this version anyway, how do I installit to turbowarp?
> 
> dude. -_- the fact that you don't know this and yet you're trying to use "this" extension is wild. i suggest getting involved with the [actual turbowarp server](https://discord.gg/jVBpHyht). in the extensions section, there is literally a BIG OL BUTTON for putting in custom extensions. ![image](https://private-user-images.githubusercontent.com/57764393/352896337-8a1ca556-7035-4db1-84c3-9e14733d0bed.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3MDIxMzgsIm5iZiI6MTcyOTcwMTgzOCwicGF0aCI6Ii81Nzc2NDM5My8zNTI4OTYzMzctOGExY2E1NTYtNzAzNS00ZGIxLTg0YzMtOWUxNDczM2QwYmVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIzVDE2NDM1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMzYzRkYmY1MzdiZWRkOTE3MjZjNzYyNWMxOGUxODZmNTA1YjdjY2YwNTQ3NGI2ZDcyOWNhNTFkZDIyMzY4YTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.BUIUvooAig3b9EBsMhtMCBxmIxKRWXtttNjZPE1vbF0)

I dont think he saw it

**Comment by glkdrlgkrlzflnjkgh** [comment]
i try to connect and auth fails EVERY TIME and i looked at documenation (well I skimmed it)
________________________________

**Reply by MikeDev101** [reply]
Check your browser's inspect console. 99% of the time it's because you didn't verify your email you used to register an account with. Did you check your email for a verification link?

**Reply by Thebloxers998** [reply]
Maybe if we don't make it send once or just add a resend email block...

**Comment by glkdrlgkrlzflnjkgh** [comment]
PLEASE ADD A RESEND EMAIL BLOCK
________________________________

**Reply by MikeDev101** [reply]
Please do not necropost. This is an in-progress feature currently being developed on the unstable server/client branch hosted on my git server: https://git.mikedev101.cc/

**Reply by Thebloxers998** [reply]
Ello!

**Reply by Thebloxers998** [reply]
Why did my translation translate it as ello to it

**Comment by Thebloxers998** [comment]
Ik how to use CLΩ but not really good at CL5

**Comment by glkdrlgkrlzflnjkgh** [comment]
oh, sorry
________________________________
